utils.h

```diff
@@ -37,6 +37,10 @@
 #define ELF_R_TYPE(val) ELF64_R_TYPE(val)
 #define ELF_R_INFO(sym, type) ELF64_R_INFO(sym, type)
 
+// From binutils
```

Comment:
I think here would be a better position for the URL?

Reply:
I see. I moved it.